Sources/IO/Geometry/STLReader/index.js

```diff
@@ -103,8 +103,14 @@ function vtkSTLReader(publicAPI, model) {
 
   // Internal method to fetch Array
   function fetchData(url, option = {}) {
-    const compression = option.compression || model.compression;
-    const progressCallback = option.progressCallback || model.progressCallback;
+    const compression =
+      option.compression !== 'undefined'
```

Comment:
why `'undefined'` and not `undefined`

Reply:
Sorry, I usually compare using `typeof` and I automatically added quotes to your snippet.